### DISCUSSION TITLE
Anyway to use spinner instead of skeleton?

### DISCUSSION THREAD

**Discussion by kellis5137** [Budibase/budibase#9470] [Help]
Anyway to use old spinner instead of the new skeleton? Without having to role back, is there anyway to force the use of spinner in version 2.1.46+ instead of skeleton loader? It it is playing chaos with some of the stuff that I've developed and I don't have time to work out the issues.

**Comment by melohagan** [comment]
Hey @kellis5137 

There had been improvements made since version **2.2.15**. Have you tried upgrading since then, or were you still seeing the same problems?

There is currently no way to select between spinner and skeleton loading in the latest version.

**Comment by kellis5137** [comment]
Hi @melohagan,
Thanks for taking a look at this! I have upgraded to 2.2.22 and still have the same problem. I added some videos in the Discord group as well as an export app as an example. You can see them here - https://discord.com/channels/733030666647765003/1069377945027739720

The table I created in the example, and in my app, is because the standard Budibase Table couldn't do what I needed. I needed to create a table with dynamic row and column names where the horizontal and vertical scaling would vary. What I did works fine for -v2.1.45, but causes the browser to crash from +v2.1.45

Definitely appreciate your input on this one!
